recipes/rb-nokogiri/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "nokogiri" %}
+{% set version = "1.10.4" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: fbf91c487d7ceb2b0593384f175843f99152322754f6ff267740bff031cead5a
+
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem --
+        --use-system-libraries
+    - gem unpack {{ name }}-{{ version }}.gem
+    - make -C {{ PREFIX }}/lib/ruby/gems/{{ ruby }}.0/gems/{{ name }}-{{ version }}/ext/{{ name }} clean
+  skip: {{ win }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - ruby
+    - libxml2
+    - libxslt
+    - rb-mini_portile2 >=2.4.0,<2.5
+    - rb-pkg-config
```

Comment:
This is the first recipe where we needed to include a ruby package during the build time. The reason is that they import [pkg-config](https://github.com/sparklemotion/nokogiri/blob/6ed19fc540b64094ef024882664fba247334614f/ext/nokogiri/extconf.rb#L119) and [mini_portile2](https://github.com/sparklemotion/nokogiri/blob/6ed19fc540b64094ef024882664fba247334614f/ext/nokogiri/extconf.rb#L468-L471) from within the [extconf.rb](https://github.com/sparklemotion/nokogiri/blob/master/ext/nokogiri/extconf.rb) file.